python/nwis_client/tests/test_nwis.py

```diff
@@ -433,7 +446,19 @@ def get_raw_mock(*args, **kwargs):
 
     monkeypatch.setattr(iv.IVDataService, "get_raw", get_raw_mock)
     df = setup_iv_value_time.get(
-        sites=["01189000"], startDt="2015-12-01T00:00", endDt="2015-12-31T23:45"
+        sites=["01189000"], startDT="2015-12-01T00:00", endDT="2015-12-31T23:45"
     )
     canonical_df = iv._create_empty_canonical_df()
     assert df.equals(canonical_df)
+
+def test_nwis_client_get_throws_warning_for_kwargs(mocked_iv):
+    from packaging import version
+    version = version.parse(nwis_client.__version__)
+    version = (version.major, version.minor)
+
+    # versions <= than 3.1 should throw an exception instead of a warning
+    assert version <= (3, 1)
+
+    with pytest.warns():
```

Comment:
I think you can specify the type of warning like this `pytest.warns(RuntimeWarning)`. I've also just learned you can do something like this to check the warning in more detail:

```python
with pytest.warns(RuntimeWarning) as record:
    warnings.warn("My RuntimeWarning Message", RuntimeWarning)

    assert len(record) == 1
    assert record[0].message.args[0] == "My RuntimeWarning Message"
```